packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -20,6 +20,79 @@ Markup: table-complex.example.html
 Style guide: components.table.complex
 */
 
+/*
+`<Table>`
+
+`Table` and sub-table components render data tables using semantic markup and ARIA attributes.
+Structure of the table and sub-table components is similar to html table elements with responsive design features such as horizontal scrolling and vertically stacked rows.
+
+@react-example Table.example.jsx
+
+@react-props Table.jsx
+
+Style guide: components.table.react
+*/
+
+/*
+`<TableBody>`
+
+A `TableBody` component renders the table body element `<tbody>` and contains a set of table rows defining the table data cell.
+
+@react-props TableBody.jsx
+
+Style guide: components.table.react-tablebody
+*/
+
+/*
+`<TableCaption>`
+
+A `TablecCaption` component specifies the `<caption>` of a table.
+
+@react-props TableCaption.jsx
+
+Style guide: components.table.react-tablecaption
+*/
+
+/*
+`<TableDataCell>`
+
+A `TableDataCell` component renders the table data cell element `<td>`.
+
+@react-props TableDataCell.jsx
+
+Style guide: components.table.react-tabledatacell
+*/
+
+/*
+`<TableHead>`
+
+A `TableHead` component renders the table head element `<thead>` and contains a set of rows defining the table header columns.
```

Comment:
`TableHead` renders the `<thead>` element and contains table rows defining the table columns.